tests/test_library.py

```diff
@@ -268,7 +269,6 @@ def test_lookup_of_station_uri(
 
             track = results[0]
             assert track.uri == station_uri.uri
-            assert next(iter(track.album.images)) == conftest.MOCK_STATION_ART_URL
```

Comment:
Why are we remove this check?  Is there another way to validate the that album image has been set properly?

Reply:
According to the mopidy [changelog](https://github.com/mopidy/mopidy/blob/develop/docs/changelog.rst#models):
"remove :attr:`mopidy.models.Album.images`"

Did I miss something?

Reply:
Oh I see. The changelog states that we should be using the new `mopidy.core.LibraryController.get_images()` method instead, and it looks like that just calls `PandoraLibraryProvider.get_images().

If you are seeing cover art images correctly during playback then we should be fine.

Reply:
Yes, images are showing up fine now.  However, I did have to make a minor change for this to work.  Chrome now blocks unencrypted content downloads under certain circumstances.  The URI returned by pydora (and I can only assume from Pandora themselves) specifies the http scheme and chrome blocks those downloads.  Therefore, in the [get_images](https://github.com/abates/mopidy-pandora/blob/416b1f881f476fffc08b9a0a19644cb7883583ae/mopidy_pandora/library.py#L140) function (see above) I am replacing http with https.  Admittedly, this feels like a bit of a hack, but I couldn't think of a better way.